polaris/viz/planar.py

```diff
@@ -36,27 +36,41 @@ def plot_horiz_field(ds, ds_mesh, field_name, out_file_name=None,  # noqa: C901
     title: str, optional
         The title of the plot
 
-    vmin, vmax : float, optional
-        The minimum and maximum values for the colorbar
+    vmin : float, optional
+        The minimum values for the colorbar
+
+    vmax : float, optional
+        The maximum values for the colorbar
```

Comment:
I think need to have separate entries in the docstring in order to format correctly in the docs.

Reply:
Yes, these parameter descriptions get duplicated in a confusing way if multiple parameters are listed together:
https://e3sm-project.github.io/polaris/main/developers_guide/generated/polaris.viz.plot_horiz_field.html#polaris.viz.plot_horiz_field